plugins/tabletools/dialogs/tableCell.js

```diff
@@ -332,97 +332,6 @@ CKEDITOR.dialog.add( 'cellProperties', function( editor ) {
 							else
 								selectedCell.removeAttribute( 'colSpan' );
 						}
-					},
-					spacer,
-					{
-						type: 'hbox',
-						padding: 0,
-						widths: [ '60%', '40%' ],
-						children: [ {
-							type: 'text',
-							id: 'bgColor',
-							label: langCell.bgColor,
-							'default': '',
-							setup: setupCells( function( element ) {
-								var bgColorAttr = element.getAttribute( 'bgColor' ),
-									bgColorStyle = element.getStyle( 'background-color' );
-
-								return bgColorStyle || bgColorAttr;
-							} ),
-							commit: function( selectedCell ) {
-								var value = this.getValue();
-
-								if ( value )
-									selectedCell.setStyle( 'background-color', this.getValue() );
-								else
-									selectedCell.removeStyle( 'background-color' );
-
-								selectedCell.removeAttribute( 'bgColor' );
-							}
-						},
-						colorDialog ? {
-							type: 'button',
-							id: 'bgColorChoose',
-							'class': 'colorChooser', // jshint ignore:line
-							label: langCell.chooseColor,
-							onLoad: function() {
-								// Stick the element to the bottom (#5587)
-								this.getElement().getParent().setStyle( 'vertical-align', 'bottom' );
-							},
-							onClick: function() {
-								editor.getColorFromDialog( function( color ) {
-									if ( color )
-										this.getDialog().getContentElement( 'info', 'bgColor' ).setValue( color );
-									this.focus();
-								}, this );
-							}
-						} : spacer ]
-					},
-					spacer,
-					{
-						type: 'hbox',
-						padding: 0,
-						widths: [ '60%', '40%' ],
-						children: [ {
-							type: 'text',
-							id: 'borderColor',
-							label: langCell.borderColor,
-							'default': '',
-							setup: setupCells( function( element ) {
-								var borderColorAttr = element.getAttribute( 'borderColor' ),
-									borderColorStyle = element.getStyle( 'border-color' );
-
-								return borderColorStyle || borderColorAttr;
-							} ),
-							commit: function( selectedCell ) {
-								var value = this.getValue();
-								if ( value )
-									selectedCell.setStyle( 'border-color', this.getValue() );
-								else
-									selectedCell.removeStyle( 'border-color' );
-
-								selectedCell.removeAttribute( 'borderColor' );
-							}
-						},
-
-						colorDialog ? {
-							type: 'button',
-							id: 'borderColorChoose',
-							'class': 'colorChooser', // jshint ignore:line
-							label: langCell.chooseColor,
-							style: ( rtl ? 'margin-right' : 'margin-left' ) + ': 10px',
-							onLoad: function() {
-								// Stick the element to the bottom (#5587)
-								this.getElement().getParent().setStyle( 'vertical-align', 'bottom' );
-							},
-							onClick: function() {
-								editor.getColorFromDialog( function( color ) {
-									if ( color )
-										this.getDialog().getContentElement( 'info', 'borderColor' ).setValue( color );
-									this.focus();
-								}, this );
-							}
-						} : spacer ]
```

Comment:
is this section removed from merging up with main repo?

Reply:
This is actual fix for the request. We have to prohibit color changes. So, we don't elements and functionality for them.